src/Sylius/Behat/Service/Converter/IriToShopPathConverter.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Behat\Service\Converter;
+
+final class IriToShopPathConverter implements IriConverterInterface
+{
+    public function convert(string $iri): string
+    {
+        return str_replace("admin", "shop", $iri);
```

Comment:
```suggestion
        return str_replace('admin', 'shop', $iri);
```

Reply:
IMHO this should be more strict to something like:
```suggestion
        return str_replace('/admin/', '/shop/', $iri);
```
Cause otherwise it can return too much if someone has custom behat rules or we add endpoint with those words inside.